lib/connect-logger.js

```diff
@@ -7,7 +7,9 @@ const levels = require('./levels');
 const DEFAULT_FORMAT = ':remote-addr - -'
   + ' ":method :url HTTP/:http-version"'
   + ' :status :content-length ":referrer"'
-  + ' ":user-agent"';
+  + ' ":user-agent"'
+  + ' ":body"';
```

Comment:
I wouldn't add body to the default log format - it is unusual for someone to want to log a request body in their http access logs. The default format is roughly equal to the standard Apache log format, so I don't want to change that.

Instead add something to the docs explaining how to add the body token to a custom format.

Reply:
As you know,
The custom token's replacement method can not access the request object.
related #717

Reply:
Please let me know if there is a way to access the request object.

Reply:
I didn't mean add "body" as a custom token, it can be one of the standard tokens but not part of the default format. So, take `:body` out of `DEFAULT_FORMAT`, but if someone wants to include the body in their log output then they can provide their own format string that includes `:body`.

Reply:
Yeah Yeah!! You're right!
I have an idea to solve this issue
I will add something to doc explaining how to use the "request body" if someone use a custom format.

I will regist another PR that i modify doc.